articles/Big Corporations and Bitcoin.txt

```diff
@@ -0,0 +1,25 @@
+|   |   |
+|---|---|
+| Title               | Could big corporations control Bitcoin? |
+| Author              | João Paulo Sant'Anna |
+| Featured text       | FIXME |
+| CT link             | FIXME |
+| CT date             | YYYY-MM-DD FIXME |
+| CT tweet            | FIXME |
+| YouTube link        | FIXME |
+| YouTube tweet       | FIXME |
+| Twitter full video  | FIXME |
+| Twitter clip 1      | FIXME |
+| Twitter clip 2      | FIXME |
+
+↑ Add more table rows for more social media links. See [this completed article](articles/simple-payment-verification.md) for a real example.
+
+Thesis statement - Will Bitcoin Succumb to Corporate Hegemony, or Will Decentralization Prevail?
+
```

Comment:
Do you have a rough bullet point outline for this article?

Reply:
Just posted the outline!